plugins/inputs/statsd/README.md

```diff
@@ -43,6 +43,7 @@
   ## Parses tags in the datadog statsd format
   ## http://docs.datadoghq.com/guides/dogstatsd/
   parse_data_dog_tags = false
+  parse_data_dog_events = false
```

Comment:
For enabling this feature, lets add a new field `datadog_extensions = false` which will control both of these, then let's deprecate the old option but if it is true we will set the new option true as well.  The new option will control both of these settings.

https://github.com/influxdata/telegraf/wiki/Deprecation#deprecate-options